randovania/game_connection/connector/remote_connector.py

```diff
@@ -94,3 +94,6 @@ async def force_finish(self) -> None:
     def is_disconnected(self) -> bool:
         """When True, this connector has lost connection with the game and must be discarded."""
         raise NotImplementedError
+
+    def inform_connected_tracker(self, tracker_details: dict | None) -> None:
+        """Called when an AutoTracker is created using this connector."""
```

Comment:
I know this is valid but I really dislike having a function with only a docstring as its contents. Would prefer an explicit `pass` statement